tests/robustness/validate/watch.go

```diff
@@ -192,24 +183,33 @@ func validateAtomic(lg *zap.Logger, report report.ClientReport) (err error) {
 }
 
 func validateReliable(lg *zap.Logger, events []model.PersistedEvent, report report.ClientReport) (err error) {
-	for _, op := range report.Watch {
-		index := 0
-		revision := firstRevision(op)
-		for index < len(events) && events[index].Revision < revision {
-			index++
-		}
-		if index == len(events) {
-			continue
+	for _, watch := range report.Watch {
+		firstRev := firstExpectedRevision(watch)
```

Comment:
is it possible to have `firstRev` > `lastRev` when there is only one response and it's progress notification?

Reply:
Yes, It's covered in test `Bookmarkable - progress notification lower than watch request - pass`.

Client can request to watch from higher revision than clusters is on. This means that progress notifies will come will lower revision than watch request.